servers/quarkus-server/src/test/java/org/projectnessie/server/TestHealthCheckNoAuthentication.java

```diff
@@ -29,9 +29,10 @@
 
 @QuarkusTest
 @TestProfile(value = TestBasicAuthentication.Profile.class)
-public class TestHealthCheckAuthentication {
+public class TestHealthCheckNoAuthentication {
```

Comment:
The intention of this test is to validate that health checks work when authentication is on. Maybe rename to `TestHealthCheckWithAuthentication`?

Reply:
Yes, that was when the management endpoints were on the same port.
But the management port has no authentication configured - that's why I renamed it, because there's no authentication.

Reply:
fair enough